src/spell.js

```diff
@@ -148,38 +148,10 @@ function parseLogicalObjectConditionValue(value) {
 /**
  * @example
  * { $or: { title: 'Leah', content: 'Diablo' } }
- * {
- *   $or: [
- *     { title: 'Leah' },
- *     { content: 'Diablo' },
- *   ],
- * }
- * {
- *   title: {
- *     $or: [
- *       'Leah',
- *       'Diablo',
- *     ]
- *   }
- * }
- * {
- *   title: {
- *     $or: [
- *       'Leah',
- *       {
- *         $like: '%jjj'
- *       },
- *     ]
- *   }
- * }
- * {
- *   title: {
- *     $not: [
- *       'Leah',
- *       'jss'
- *     ]
- *   }
- * }
+ * { $or: [ { title: 'Leah' }, { content: 'Diablo' } ] }
+ * { title: { $or: [ 'Leah', 'Diablo' ] } }
+ * { title: { $or: [ 'Leah', { $like: '%jjj' } ] } }
+ * { title: { $not: [ 'Leah', 'jss' ] } }
```

Comment:
changed into a format that is more compact, like the output of `util.inspect(obj)`